src/masternodeman.h

```diff
@@ -129,6 +129,7 @@ class CMasternodeMan
     // these maps are used for masternode recovery from MASTERNODE_NEW_START_REQUIRED state
     std::map<uint256, std::pair< int64_t, std::set<CNetAddr> > > mMnbRecoveryRequests;
     std::map<uint256, std::vector<CMasternodeBroadcast> > mMnbRecoveryGoodReplies;
+    std::vector< std::pair<CService, uint256> > vScheduledMnbRequestConnections;
```

Comment:
You shouldn't use a std::vector as FIFO, it's very inefficient because you need to move all the elements on pop, should use a std::list (or better std::queue for cleaner interface) instead.